libpod/container_internal.go

```diff
@@ -1069,7 +1069,7 @@ func (c *Container) generatePasswd() (string, error) {
 	}
 	originPasswdFile := filepath.Join(c.state.Mountpoint, "/etc/passwd")
 	orig, err := ioutil.ReadFile(originPasswdFile)
-	if err != nil {
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
We still want the passwd created so that getpwuid will work for the current uid.  All that happens here is an /etc/passwd file gets created with one line, for the user that is running.

Reply:
Thanks, Dan. I missed that entirely.